Sources/PostgresNIO/Postgres+PSQLCompat.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  Postgres+PSQLCompat.swift
+//  
+//
+//  Created by Fabian Fett on 19.01.21.
+//
+
+struct PostgresJSONDecoderWrapper: PSQLJSONDecoder {
```

Comment:
Ditch Xcode template headers.
```suggestion
struct PostgresJSONDecoderWrapper: PSQLJSONDecoder {
```